bucket/imagemagick.json

```diff
@@ -3,17 +3,20 @@
     "description": "Create, edit, compose, and convert 200+ of bitmap images formats.",
     "homepage": "https://imagemagick.org/",
     "license": "ImageMagick",
-    "depends": "ffmpeg",
+    "suggest": {
+        "ffmpeg": "ffmpeg"
+    },
     "architecture": {
         "64bit": {
-            "url": "https://download.imagemagick.org/ImageMagick/download/binaries/ImageMagick-7.1.0-portable-Q16-HDRI-x64.zip",
-            "hash": "b9624838865c747e810e19da7f1a9ba915aed93cbbad2dc4615251e28aa9e649"
+            "url": "https://www.imagemagick.org/download/binaries/ImageMagick-7.1.0-13-Q16-HDRI-x64-dll.exe",
+            "hash": "c4607f68794f9e8e5ed73b1c0a295aae8e612c4b63b163e65c5f2669068b78d0"
         },
         "32bit": {
-            "url": "https://download.imagemagick.org/ImageMagick/download/binaries/ImageMagick-7.1.0-portable-Q16-HDRI-x86.zip",
-            "hash": "366bab6ef6166412f2a456177026bb93202e265d0399314ffdc4820af277a065"
+            "url": "https://www.imagemagick.org/download/binaries/ImageMagick-7.1.0-13-Q16-HDRI-x86-dll.exe",
+            "hash": "477bf261a8ad6d3034e0b6f58ce7abdeb9a2c6186c9caef34e64fe9e24eafa42"
         }
     },
+    "innosetup": true,
     "bin": [
```

Comment:
Remove the "convert.exe" from the bins to prevent conflict with Windows' internal convert utility. Add a note that ImageMagick's convert has been aliased as "imconvert"

Reply:
Done!